devon4j-http-rest-client/index.asciidoc

```diff
@@ -0,0 +1,126 @@
+createFile("httprestserver/api/src/main/java/com/example/application/httprestserver/visitormanagement/service/api/rest/VisitormanagementRestService.java", "files/VisitormanagementRestService.java")
+createFile("httprestserver/core/src/main/java/com/example/application/httprestserver/visitormanagement/service/impl/rest/VisitormanagementRestServiceImpl.java", "files/VisitormanagementRestServiceImpl.java")
+changeFile("httprestserver/core/src/main/resources/application.properties", { "file": "files/server_application.txt" })
+changeFile("httprestserver/core/src/main/resources/config/application.properties", { "file": "files/server_config_application.txt" })
+--
+Here , you can see "VisitormanagementRestServiceImpl.java" is annotated with @Named to make it a spring-bean. To get return response to client "returnResponseToClient()" can be accessed via HTTP GET under the URL path "/visitormanagement/v1/clientrequest". It will return its result (String) as JSON (see @Produces in VisitormanagementRestService).
+====
+
+
+====
+You have successfully built the REST-server. Now, you have to start the build and then, start the server as mentioned below.
+[step]
+--
+buildJava("httprestserver", false)
+runServerJava("httprestserver/server", { "startupTime": 1000, "port": 8080, "path": "httprestserver" })
+nextKatacodaStep(" ", [{ "file": "files/viewPort.txt" }])
+--
+In next step, You have to create devon4j service Client.
+====
+
+
+====
+To create devon4j service Client. First, You need to create a sample devon4j project.
+
+[step]
+== Create devon4j service Client
+--
+createDevon4jProject("com.sample.application.httprestclient")
+--
+Then, You need to add required dependency to your application. 
+====
+
+
+[step]
+--
+nextKatacodaStep("Dependency Setup ", [{ "file": "files/dependency.txt" }])
+--
+
+====
+You have to create a Java interface VisitormanagementRestService to invoke.
+You have to create rest client i.e. Devon4jRestClient.java and it's implementation class i.e. Devon4jRestClientImpl.java.
+[step]
+== Create devon4j rest client
+--
+createFile("httprestclient/api/src/main/java/com/sample/application/httprestclient/general/service/api/rest/VisitormanagementRestService.java", "files/VisitorManagementRestServiceClient.txt")
+createFile("httprestclient/api/src/main/java/com/sample/application/httprestclient/general/service/api/rest/Devon4jRestClient.java", "files/Devon4jRestClient.java")
+createFile("httprestclient/core/src/main/java/com/sample/application/httprestclient/general/service/impl/rest/Devon4jRestClientImpl.java", "files/Devon4jRestClientImpl.java")
+nextKatacodaStep(" ", [{ "file": "files/explainSynchronousCall.txt" }])
+--
+After getting response, you can handle the response further in your implementation. Here, you can see that response is getting handled in "showResponse" method.
+====
+
+
+====
+# Configuration 

```

Comment:
Add a # or use == for this header. This would make the configuration chapter header the same size as the tutorial header.

Reply:
I have modified # to ==.